spec/template.dd

```diff
@@ -1082,6 +1082,23 @@ $(H2 $(LNAME2 function-templates, Function Templates))
         }
         ---
 
+    $(P Variadic Function Templates can have parameters with default values.
+        These parameters are always set to their default value in case of IFTI.
+    )
+
+        $(SPEC_RUNNABLE_EXAMPLE_COMPILE
+        ---
+        size_t fun(T...)(T t, string file = __FILE__)
+        {
+            import std.stdio;
+            writeln(file, " ", t);
+            return T.length;
+        }
+        assert(fun(1, "foo") == 2);  // not 1 (IFTI)
+        assert(fun!int(1, "foo") == 1);  // IFTI
```

Comment:
Hmm the comments here don't make much sense anymore. I suggest we either remove them or use a better explanation. What do you think?

Reply:
(updated, after discussed on slack)